tsconfig.json

```diff
@@ -1,13 +1,14 @@
 {
   "compilerOptions": {
+    "rootDir": ".",
     "baseUrl": "./",
     "importHelpers": true,
     "outDir": "./dist/out-tsc",
     "strict": true,
     "sourceMap": true,
     "declaration": false,
     "module": "es2020",
-    "target": "es2015",
+    "target": "es2020",
```

Comment:
Is changing the target required? Just curious why we're doing it now

Reply:
Would have to go through compatibility - it doesn't really matter to us since TS handles the down-compilation, so no harm (other than missed optimizations) in targeting the older version with more compatibility

Reply:
In that case, what about `.browserlistrc`? Just by virtue of updating Angular, we are limiting browser support to the most recent browsers at the time of Angular 14's release (Q2 2022). (See [browserlist](https://github.com/browserslist/browserslist/blob/main/README.md)).

IMO we should update this to ES2021 or ES2020, and we should check for compatibility after updating Angular anyways, regardless of any internal SLA that might be different than Angular's (which is "[the most recent browsers](https://angular.io/guide/browser-support)", presumably when Angular 14 was released last year).

Reply:
I think we can decouple this one with the current PR. I will do some research and take your concern into account and put a new PR.

Reply:
Agree, decouple. There's a difference in angular (and our) published SLA and best effort. Like there's no way angular _actually_ only supports the last 2 versions of chrome (i.e. the march and april releases, but not feb), or other evergreens. That's just what they're putting in writing. We've got to be careful with older support given the nature of larger enterprise users and slower upgrade cadences.